modules/main.js

```diff
@@ -61,11 +29,11 @@ class Book {
     this.books.forEach((book, index) => {
       const div = document.createElement('div');
       div.innerHTML = `
-       &nbsp ${book.title} &nbsp By 
-                ${book.author}
-                <button class="remove-button" data-index="${index}">Remove</button>
-                <br><br>
-            `;
+         &nbsp ${book.title} &nbsp By 
+                  ${book.author}
+                  <button class="remove-button" data-index="${index}">Remove</button>
+                  <br><br>
+              `;
```

Comment:
- [ ] in order to import the class as you are supposed to export it as well please make sure to export the class correctly.